runtime/compiler/env/j9method.cpp

```diff
@@ -6689,15 +6699,12 @@ bool
 TR_ResolvedJ9Method::shouldCompileTimeResolveMethod(I_32 cpIndex)
    {
    I_32 realCPIndex = jitGetRealCPIndex(_fe->vmThread(), romClassPtr(), cpIndex);
```

Comment:
This should be moved to `getMethodNameAndSignatureFromConstantPool`, and pass the cpIndex from bytecode to `getMethodNameAndSignatureFromConstantPool`

Reply:
Done.